tests/src/test/java/org/eclipse/hono/tests/amqp/CommandAndControlAmqpIT.java

```diff
@@ -648,16 +755,82 @@ public void testSendCommandFailsWhenNoCredit(
                         assertThat(((ServerErrorException) t).getErrorCode()).isEqualTo(HttpURLConnection.HTTP_UNAVAILABLE);
                         // with no explicit credit check, the AMQP adapter would just run into the "waiting for delivery update" timeout (after 1s)
                         // and the error here would be caused by a request timeout in the sendOneWayCommand() method above
+                        assertThat(t).isNotInstanceOf(SendMessageTimeoutException.class);
```

Comment:
can we then remove the assertion below?

Reply:
Not yet, the `org.eclipse.hono.client.amqp.RequestResponseClient` doesn't use a `SendMessageTimeoutException` yet. I'll have a look at that after this PR.